integration-build/action.yml

```diff
@@ -0,0 +1,37 @@
+name: "Integration Build"
+description: "Build using Maven"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout branch
+      uses: actions/checkout@v3
+
+    - name: Set up Java 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        server-id: github
+
+    - name: Cache dependencies
+      uses: actions/cache@v3
+      with:
+        path: ~/.m2
+        key: v1-dependencies-{{ checksum "pom.xml" }}
+        restore-keys: v1-dependencies-
+
+    - name: Maven go offline
+      shell: bash
+      run: mvn -s $GITHUB_WORKSPACE/.github/settings.xml dependency:go-offline
```

Comment:
Minor think that propably never happen, but this could be "$GITHUB_WORKSPACE/.github/settings.xml" so if there is space in in workspace path.

Reply:
Good point. Added quotation marks to filepath.